modules/functions.bash

```diff
@@ -1,6 +1,6 @@
 export HOMEBREW_CASK_OPTS="--appdir=/Applications"
```

Comment:
Also, please change back permissions of this file.

Reply:
You about RBENV_ROOT and NVM_DIR?

Reply:
No, I'm talking about file permissions
![image](https://user-images.githubusercontent.com/57901245/128686914-5cb695e7-98ec-408d-be8a-a95187709905.png)

Reply:
Please help me, how I can do this?

Reply:
`chmod 644 modules/functions.bash`

Reply:
Thank you, but I don't remember that I chmod for this file, hmm